pubspec.yaml

```diff
@@ -17,3 +17,49 @@ dependencies:
 dev_dependencies:
   pedantic: ^1.0.0
   test: ^1.0.0
+
+dependency_overrides:
```

Comment:
@natebosch @jakemac53 – are we still doing overrides or should we just set the dependency constraint to pull in the published packages?

Reply:
pull in the published ones where you can imo